AirCasting/AuthViews/CreateAccountView.swift

```diff
@@ -37,63 +37,67 @@ struct CreateAccountView: View {
 
 private extension CreateAccountView {
     var contentView: some View {
-        GeometryReader { geometry in
-            ScrollView {
-                VStack(spacing: 50) {
-                    if lifeTimeEventsProvider.hasEverLoggedIn {
-                        progressBar.hidden()
-                    } else {
-                        progressBar
-                    }
-                    titleLabel
-                    VStack(spacing: 20) {
-                        VStack(alignment: .leading, spacing: 5) {
-                            emailTextfield
-                                .keyboardType(.emailAddress)
-                            if !isEmailCorrect {
-                                errorMessage(text: AuthErrors.incorrectEmail.localizedDescription)
-                            }
+        ZStack(alignment: .bottomTrailing) {
+            Image("dashboard-background-thing")
```

Comment:
This image goes uo with the kayboard, please have this in mind. IDK what is better here